dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/user/DefaultUserService.java

```diff
@@ -105,20 +109,24 @@
 
     private final SessionRegistry sessionRegistry;
 
+    private final SecurityService securityService;
+
     private final Cache<String> userDisplayNameCache;
 
     public DefaultUserService( UserStore userStore, UserGroupService userGroupService,
         UserRoleStore userRoleStore,
         CurrentUserService currentUserService, SystemSettingManager systemSettingManager,
         CacheProvider cacheProvider,
-        @Lazy PasswordManager passwordManager, @Lazy SessionRegistry sessionRegistry )
+        @Lazy PasswordManager passwordManager, @Lazy SessionRegistry sessionRegistry,
```

Comment:
Is this lazy to due circular dependencies? If so, would be good to resolve them properly.